### ISSUE TITLE
Don't quite understand the section "Don't write to global functions"

### ISSUE THREAD

**Issue by panigrc** [jupeter/clean-code-php#96]
I don't understand how the 

```php
// config.php
return [
    'foo' => 'bar',
];
```

How is the array being used ?

[link](https://github.com/jupeter/clean-code-php#dont-write-to-global-functions)

**Comment by peter-gribanov** [comment]
I agree. It doesn't look very clear.
The idea was this:

```php
$config = include 'config.php';

$configuration = new Configuration($config);
```

**Comment by panigrc** [comment]
Ok Thanks for clarifying that !  :+1:

**Comment by peter-gribanov** [comment]
@panigrc Please take a look, my PR #97.
This example became more clear?

**Comment by TomasVotruba** [comment]
Resolved by #97
